include/boost/geometry/algorithms/detail/equals/implementation.hpp

```diff
@@ -138,32 +137,28 @@ struct area_check
                              Geometry2 const& geometry2,
                              Strategy const& strategy)
     {
-        return geometry::math::equals(
-            geometry::area(geometry1,
-                           strategy.template get_area_strategy<Geometry1>()),
-            geometry::area(geometry2,
-                           strategy.template get_area_strategy<Geometry2>()));
+        return geometry::math::equals(geometry::area(geometry1, strategy),
+                                      geometry::area(geometry2, strategy));
     }
 };
 
 
+/*
```

Comment:
so this is not needed? do we need it for future use?

Reply:
I don't know it seems not. I probably noticed that this is not used anywhere but was not sure. So in case I'm mistaken I simply comented it out instead of removing it.

Reply:
OK